methods/data_cleaning.py

```diff
@@ -15,3 +15,30 @@ def drop_duplicates(df):
         df.reset_index(inplace=True, drop=True)
     df.head(5)
     return df
+
+
+def make_columns_python_friendly(df):
```

Comment:
will this function have another use cases? If not, it seems to me, that a single function 'clean_column_names(df)' seems to be enough and would explain what this function is for (rather than 'make_columns_python_friendly').

Reply:
something like this ist in my mind but all your nice extras:
```python
def rename_columns(df):
    df.columns = df.columns.str.lower().str.replace(" ", "_")
    return df
```

Reply:
The function can be renamed to that of course.
It just also has 2 more functions
deleting special characters from the columns names like "§$%&" and adding an Underscore when the column starts with a number

We can make it cleaner or shorter of course =)

Reply:
I realized that your functions does more. I you agree on my suggestion, change it. Just push another commit with the changes.